cmd/syncthing/gui.go

```diff
@@ -125,11 +120,13 @@ func (s *apiSvc) getListener(cfg config.GUIConfiguration) (net.Listener, error)
 		},
 	}
 
-	rawListener, err := net.Listen("tcp", cfg.Address)
+	rawListener, err := net.Listen("tcp", guiCfg.Address())
 	if err != nil {
 		return nil, err
 	}
 
+	l.Infoln("Starting web GUI on", guiCfg.URL())
```

Comment:
That was the case before also. There's another "API listening on ...", and I'm not sure we really need both, but this one does print the URL and the other prints the listen address...